scripts/maintenance/migrate_project/README.md

```diff
@@ -1,34 +1,51 @@
 # project migration
 
 Built on top of the existing `postgres-database` package.
-It is used to migrate a user's project and eventually (hidden projects, generated via the api).
 
-If a file's or project's unique identifier already exist in the destination database, the process will not continue.
+It is used to migrate a user's project. Currently, this does not work for hidden projects, generated via the api.
 
-**NOTE:** data sync is done using `rclone`, currently it is assumed that the data source is a `MINIO S3 backend` and the destination is an `AWS S3 backend`.
+It uses low-level direct access to pgSQL and S3. Data sync is done using `rclone`. Therefore, this script might become outdated upon database changes in osparc-simcore.
 
 
-Any doubts? Ask **ANE**.
+# IMPORTANT PITFALLS:
+- Currently, this does not work for hidden projects, generated via the api.
+- No postgres database version migration is performed atm. This migration only works for identical source and target RDBs.
+- If a file's or project's UUID already exist in the destination database (colision), this script will fail with an error.
+- Supported S3 providers are `CEPH`, `AWS`, `MINIO`
+
+
+# Maintainers:
+ANE, DK
+
 # How to use
 
-Build the image locally
+1. Build the image locally
 
 ```
 make build
 ```
 
-Create a configuration file 
+Create a configuration file
 
 ```
-make empty-config-file 
+make create-empty-config-file
```

Comment:
this recipe's name is a bit weird together with make. I guess ```make empty-config-file``` or ```make config-file``` makes more sense. It could also just be an automatic recipe that will run only if the config does not exist.

Reply:
Renamed, see Perdos comment